addons/web/static/src/boot.js

```diff
@@ -440,4 +427,42 @@
         // visit each root to find cycles
         return visitJobs(jobs.map((j) => j.name));
     }
+
+    function displayModuleErrors({ failed, missing, unloaded, cycle }) {
+        const esc = (str) => {
+            const t = document.createElement("t");
+            t.innerText = str;
+            return t.innerHTML;
+        };
+        const ul = (heading, arr) =>
+            arr && arr.length
+                ? `${heading} <ul>${arr.map((el) => `<li>${esc(el)}</li>`).join("")}</ul>`
+                : "";
+        if ([failed, missing, unloaded].some((arr) => arr.length) || cycle) {
+            document.body.innerHTML = `
+            <div class="position-fixed w-100 h-100 d-flex align-items-center flex-column bg-white overflow-auto" role="alert" style="z-index: 10000">
```

Comment:
10001 would be safer I think

Reply:
Basically we want this to be on top of everything else, there are a few things that are above 1000 in bootstrap, this is a way to go above them. Because while we replace the content of the document body, JS is running and the webclient may be mounted anyway, but we don't want it to be visible or to be able to interact with it.